core/test/com/google/inject/MethodInterceptionTest.java

```diff
@@ -269,6 +270,11 @@ protected void configure() {
 
   @Test
   public void testNotInterceptedMethodsInInterceptedClassDontAddFrames() {
+    // Test relies on Enhancer appearing in stack traces which isn't true for Java17 hidden classes
```

Comment:
can we scope this around the ==true portion, so that the ==false portion is still tested?

alternately, can we add a java17+ test that the hidden $Enhancer classes never appear in stack traces so that the same property is tested (but perhaps in a broader manner now -- incl that intercepted methods don't get it) ?

same thing w/ the FastClass stacks below.

Reply:
Managed to fix this another way: 385c857e8